styleguide/source/_patterns/02-organisms/06-topic/04-topic-related-content/topic-related-content.twig

```diff
@@ -0,0 +1,4 @@
+<div class="topic_related-content">
+  {% include 'atoms-h5' with { 'content': 'Lorem ipsum', 'class': 'topic_related-content_title' } %}
```

Comment:
Suggestion to use existing atom `atoms-title-label` here. UX would like an `<h3>` for the element, and we're already using it elsewhere in the template